1.adoc

```diff
@@ -21,6 +21,15 @@ Platforma:: Webový prohlížeč (editor je komponenta použitelná v rámci web
     * co dělal...
 
 == Product statement
+
+Markup editor je webová embedovatelná komponenta pro editaci dokumentů v jazycích LML.
+Editor je určen pro technicky zdatné uživatele, u kterých se předpokládá
```

Comment:
Tohle mi připadá frázovaný spíš jako Restrictions

možná spíš něco jako "Editor staví na znalostech a zkušenostech uživatelů s LML a IDE, a poskytuje jim vhodné nástroje pro co nejlepší využití těchto znalostí."

Reply:
Nemyslim si, je to jen predpoklad o uzivatelich. Stejne jako kdybys navrhoval IDE tak si reknes ze uzivatele jsou programatori. Tim padem pak nebudes testovat jestli v nem umi _aspon neco_ rybar pri uzivatelskym testovani, ale rybare vyhodis, ze je ti k nicemu, protoze neni typickej uzivatel.

Myslim, ze je naopak dobry si cilovou skupinu uzivatelu explicitne definovat.

Reply:
Pokud to chces takhle explicitne, tak proc ne.